x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/action/ResumeFollowAction.java

```diff
@@ -266,6 +266,8 @@ public void readFrom(final StreamInput in) throws IOException {
             maxReadRequestOperationCount = in.readOptionalVInt();
             maxOutstandingReadRequests = in.readOptionalVInt();
             maxReadRequestSize = in.readOptionalWriteable(ByteSizeValue::new);
```

Comment:
Would you mind moving maxReadRequestSize above maxOutstandingReadRequests so that we have the same order for both read and write.